packages/calendar/examples/calendar/pages/demo/demo.page.ts

```diff
@@ -1,10 +1,46 @@
 import { Component } from '@angular/core';
+import { DateRange } from '@acpaas-ui/js-date-utils';
 
 @Component({
 	templateUrl: './demo.page.html',
 })
 export class DemoPageComponent {
+	public selectedDate: Date = new Date();
+	public range: DateRange = [1, 6];
+
+	public javascript1 = `import { CalendarModule } from '@acpaas-ui/ngx-components/calendar'
```

Comment:
Wow dit lijst héél raar! 😄 Kunnen we die snippets niet op een meer leesbare manier toevoegen?

Reply:
Ik hoor graag hoe 😁

Reply:
Kunnen we de code-snippet component aanpassen zodat je de snippet in de template zelf kan invullen? Hoeft zeker niet in deze PR maar misschien moeten we het wel toevoegen als een issue.

Reply:
of we moeten meer naar een styleguidist systeem met een markdown file die wordt uitgelezen?

Reply:
Het was de bedoeling dat we jullie nog eens contacteerden daarover, maar goed dat je het zelf voorstelt! 👍